### DISCUSSION TITLE
Replay to email breaks existing thread (USA way - latest first)

### DISCUSSION THREAD

**Discussion by marek141017** [tutao/tutanota#6400] [Bugs]
Every time I replay to someones email (no matter what it is replay or new), it breaks email thread and web interface shows new message on the left side. When I open it I can see all messages I have sent and received inside + new replay. As a result I have to delete all previously created threads to keep only one.

P.S.
Just checked mobile version and it behaves the same way – replay goes to new thread. 
So now I think it is not a front-end issue, it is more likely back-end issue

Thank you in advance

**Comment by ganthern** [comment]
Hello there :wave:

I assume you're talking about the new entry that appears in the mail list on the left?  We're currently showing each mail separately in the lists, even though they may belong to the same conversation thread. Deleting them will move them to trash and eventually make them disappear completely, even from the thread in the conversation viewer on the right.

If your problem is with something else (I may be misunderstanding), could you provide screenshots that demonstrate the issue?

**Reply by marek141017** [reply]
Okay I see. It is feature not a bug.
Do you realize that your service multiply email content this way. If I got 100 replies it creates 100 threads with attachments (if any). Plus it is utterly annoying – every time one needs to delete previous thread in order to reply into the last.

**Reply by ganthern** [reply]
Replying to the last mail should be possible automatically, they're sorted newest to oldest. Clicking any mail in the list and using the R shortcut / using the reply button should open a mail editor with a fresh reply to exactly that mail. It's a bug if that is not the case.

We're aware that having the "Thread" or conversation view on the right and individual e-mails on the left is unusual, but since implementing conversations in the list is a separate, significant work item we opted to wait for feedback on it.

With regard to the data duplication you can rest assured that it's not an issue; the mail view would have to load the same data either way to display everything in the right order.
